x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -337,3 +338,15 @@ export const routes: APMRouteDefinition[] = [
     ),
   },
 ];
+
+function withApmServiceContext(
+  WrappedComponent: (props: RouteComponentProps<any>) => JSX.Element | null
```

Comment:
I think this is simpler.

```suggestion
  WrappedComponent: React.ComponentType<any>
```